assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -1,15 +1,15 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 
 name: convert_model_to_mlflow
-version: 0.0.7
+version: 0.0.8
 type: command
 
 is_deterministic: True
 
 display_name: Convert Hugging face models to MLflow
 description: Component converts Huggingface models to MLflow model packaging format
 
-environment: azureml://registries/azureml/environments/model-management/versions/10
+environment: azureml://registries/azureml/environments/model-management/versions/11
```

Comment:
This needs to change as well after the new version gets pushed to PyPi isn't it?

Reply:
yes, incremented the env version to make sure we pick the right env after release